### PR TITLE
Resolve 5.4.0 warnings

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

Upgrades polkadot.js to `9.14.2`. I tried with a newer 10 version, but that seems to have changed how the types are generated, which is likely a large change for us. 9.14.2 seems to be OK and gets rid of the substrate warnings for undecorated types.

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

**Comment by polymath-eric** [comment]
This introduces some new warnings: 
```
2023-04-20 16:30:09        API/INIT: polymesh_dev/5004000: Not decorating unknown runtime apis: PipsApi/1, ProtocolFeeApi/1, IdentityApi/2, AssetApi/2, 0x9f242c2c886f3fce/1, GroupApi/1, 0x9ea061a615cee2fe/1
```

**Comment by polymath-eric** [comment]
A strange test failure. they pass on my machine: https://github.com/PolymeshAssociation/polymesh-sdk/actions/runs/4758601724/jobs/8456981466

File: package.json [hunk 1]
```diff
@@ -32,8 +32,8 @@
     "generate:meta": "ts-node --skip-project node_modules/.bin/polkadot-types-from-chain --endpoint metadata.json --package polymesh-types --output ./src/polkadot --strict",
     "generate:consts": "cross-env node scripts/generateConsts.js",
     "generate:post-process": "cross-env node scripts/postProcessTypes.js",
-    "test": "jest --detectOpenHandles --coverage",
```
Comment (polymath-eric) [inline package.json:35]: https://jestjs.io/docs/cli#--detectopenhandles

Comment (polymath-eric) [reply package.json:35]: https://stackoverflow.com/questions/68012814/gitlab-jest-test-segmentation-fault-core-dumped

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**